apps/codeforafrica/src/payload/blocks/GetInvolved.js

```diff
@@ -0,0 +1,17 @@
+import impacts from "../fields/impacts";
+import linkGroup from "../fields/links/linkGroup";
+
+const GetInvolved = {
+  slug: "get-involved",
+  imageURL: "/images/cms/blocks/get_involved.jpg",
+  imageAltText: "Our Impact Statisctics",
```

Comment:
```suggestion
  imageAltText: "Our impact in summary",
```

Or just use the name i.e. `imageAltText: "Get Involved"`